tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/TspTool/TspTool.cs

```diff
@@ -19,11 +19,21 @@ namespace Azure.Sdk.Tools.Cli.Tools
     public class TypeSpecTool(INpxHelper npxHelper, ILogger<TypeSpecTool> logger, IOutputService output) : MCPTool
     {
 
+        // This is the template registry URL used by the TypeSpec compiler's init command.
+        private const string AzureTemplatesUrl = "https://aka.ms/typespec/azure-init";
+
         // commands
         private const string ConvertSwaggerCommandName = "convert-swagger";
+        private const string InitCommandName = "init";
 
+        // Shared command options
         private readonly Option<string> outputDirectoryArg = new("--output-directory", "The output directory for the generated TypeSpec project. This directory must already exist and be empty.") { IsRequired = true };
 
+        // Init command options
+        private readonly Option<string> templateArg = new("--template", "The template to use for the TypeSpec project. Valid values are: azure-core (for data-plane services), azure-arm (for resource-manager services)") { IsRequired = true };
```

Comment:
Using an enum type here instead of `string` would validate inclusion within the limited options.

Reply:
Updated to an enum, though I'm a little sad I still have to use a string with the choices listed out for the MCP path 😢